.eslintrc

```diff
@@ -6,7 +6,6 @@
 		"node": true
 	},
 	"rules": {
-		"quotes": ["error", "double"],
```

Comment:
Probably for a different time, but how about `["error", "double", { "avoidEscape": true }]`?